src/main.rs

```diff
@@ -25,7 +25,7 @@ pub static CONFIG: Config = Config {
 	window_height: 720.0,
 	window_bottom_y: 720.0 / -2.0,
 	window_left_x: 1024.0 / -2.0,
-	floor_thickness: 10.0,
+	floor_thickness: 500.0,
```

Comment:
Change that back after you tested the walking with camera... not sure why the cat keeps falling through the solid body...